src/dasi/tools/DASITool.cc

```diff
@@ -16,7 +16,7 @@ static DASITool* instance_ = nullptr;
 DASITool::DASITool(int argc, char** argv): eckit::Tool(argc, argv) {
     ASSERT(instance_ == nullptr);
     instance_ = this;
-    options_.push_back(new SimpleOption<std::string>("config", "DASI Configuration file (yaml)"));
+    options_.push_back(new eckit::option::SimpleOption<std::string>("config", "DASI Configuration file (yaml)"));
```

Comment:
You shouldn't need the eckit::option:: here - you use the namespace higher up in the file!